#! /usr/bin/env python

PACKAGE='synthetic2d'

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add('score_thresh', double_t, 0, 'Score threshold of the instance segmentation output', min=0., default=0.7, max=1.)
gen.add('nms_thresh', double_t, 0, 'NMS threshold of the instance segmentation output', min=0., default=0.3, max=1.)
gen.add('context', str_t, 0, 'List of objects', default='')
gen.add('target', int_t, 0, 'Target object label', min=-1, default=-1)

exit(gen.generate(PACKAGE, PACKAGE, 'MaskRCNNRelook'))
